utils/paths.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+/**
+ * Path to the temporary local image folder
+ */
+const STORAGE_IMG_DIR = '.sketch-constructor/images';
```

Comment:
I like how this path is based on the current working directory rather than some global path like `~/.sketch-constructor`!

Reply:
Yep, I also think thats quite neat.